tests/unit/backend/layers/common/base_api_test.py

```diff
@@ -24,6 +24,10 @@ def setUp(self):
         )
         self.mock_config.start()
 
+        # TODO: this can be improved, but the current authorization method requires it
+        self.mock = patch("backend.common.corpora_config.CorporaAuthConfig.__getattr__", return_value="mock_audience")
+        self.mock.start()
+
```

Comment:
Moved here, to the narrowest class necessary. Only required for auth tests, which inherit from this class.